exercises/practice/custom-set/.meta/config.json

```diff
@@ -1,6 +1,47 @@
 {
   "blurb": "Create a custom set type.",
   "authors": [],
```

Comment:
Another empty one, added in https://github.com/exercism/elixir/commit/fbbfd1c4f0c4ce936b866f66a2f68cedf9a030cc